src/rules/no-invalid-element-name.ts

```diff
@@ -5,18 +5,29 @@
 
 import {Rule} from 'eslint';
 import * as ESTree from 'estree';
-import validate = require('validate-element-name');
+import isValidElementName = require('is-valid-element-name');
 
 //------------------------------------------------------------------------------
 // Rule Definition
 //------------------------------------------------------------------------------
 
+export const errorMessage =
+  "Element name is invalid and should follow the HTML standard's recommendations (https://html.spec.whatwg.org/multipage/custom-elements.html#prod-potentialcustomelementname).";
```

Comment:
just for code readability can you split this over two lines?

```ts
export const errorMessage = "Element name is invalid and should follow the HTML standard's recommendations" +
  " (URL_HERE)";
```

Reply:
Cool, I've reformatted this